src/main/sql/example_data.sql

```diff
@@ -1,20 +1,29 @@
 --
--- Example Client, please sync to the attribute values in the addon-administration.properties.
--- Please update all queries in this file to your needs!
--- Have to be imported in the database of the auth-server, before you deploy the addon-administration!
+-- DEPRECATED: replaced by extension.sql and admin_group.sql
 --
--- resource server: example group 'admin' for the addon-administration
+
+INSERT INTO osiam_client (internal_id, accesstokenvalidityseconds, client_secret, expiry,
+                          id, implicit_approval, redirect_uri, refreshtokenvalidityseconds,
+                          validityinseconds)
+VALUES (20, 28800, 'super-secret', NULL,
+        'addon-administration-client', TRUE, 'http://localhost:8080/addon-administration', 86400,
+        0);
+
+INSERT INTO osiam_client_scopes (id, scope) VALUES (20, 'GET');
+INSERT INTO osiam_client_scopes (id, scope) VALUES (20, 'POST');
+INSERT INTO osiam_client_scopes (id, scope) VALUES (20, 'PUT');
+INSERT INTO osiam_client_scopes (id, scope) VALUES (20, 'PATCH');
+INSERT INTO osiam_client_scopes (id, scope) VALUES (20, 'DELETE');
+INSERT INTO osiam_client_grants (id, grants) VALUES (20, 'authorization_code');
 
 INSERT INTO scim_meta (id, created, lastmodified, location, resourcetype, version)
 VALUES (2, '2011-10-10', '2011-10-10', NULL, 'Group', NULL);
 
 INSERT INTO scim_id (internal_id, external_id, id, meta_id)
-VALUES (2, 'group_external_id', 'cef9452e-00a9-4cec-a086-d171374aabef', 2);
+VALUES (2, NULL, 'cef9452e-00a9-4cec-a086-d171374aabef', 2);
```

Comment:
I don't know what this means, but AFAIC the external id should be empty, as it is controlled by the service consumer not the service provider.